icon-stack/client-apis/json-rpc-api/v3.md

```diff
@@ -1901,7 +1901,7 @@ Response on success
 | p2pEndpoint | String     |     O    | Network info used for connecting among P-Rep nodes "123.45.67.89:7100", "node.example.com:7100"               |
 | nodeAddress | String     |     X    | Node Key for only consensus "hxe7af5fcfd8dfc67530a01a0e403882687528dfcb"                                      |
 
-\*details : See [JSON Standard for P-Rep Detailed Information](broken-reference)
+\*details : See [JSON Standard for P-Rep Detailed Information](https://docs.icon.community/v/icon1/references/reference-manuals/json-standard-for-p-rep-detailed-information)
```

Comment:
None of the ICON v1 docs should be referenced from the ICON v2 docs. ICON v2 docs should be self-contained. I would recommend adding the JSON standard for validators as a subsection under the following:

ICON Stack -> Client APIs -> JSON-RPC API -> <new section on JSON standards for validator nodes>

Reply:
thats seems fairly easy to do, I can update the pull request to add those changes, my question is, how can I paste in this document the link of a document that is not currently online? because If I create the `json-standard-for-validators.md` inside `icon-stack > client-apis > json-rpc-api` there is no public link for it (like docs.icon.community/something-else) to paste in here for the broken links.

it seems that first the `json-standard-for-validators.md` gotta be created, pushed and publish before fixing the broken links in here